cocotb/share/lib/simulator/simulatormodule.cpp

```diff
@@ -1071,6 +1100,7 @@ template<>
 PyTypeObject gpi_hdl_Object<gpi_iterator_hdl>::py_type = []() -> PyTypeObject {
     auto type = fill_common_slots<gpi_iterator_hdl>();
     type.tp_name = "cocotb.simulator.gpi_iterator_hdl";
+    type.tp_doc = "GPI object sub-object iterator.";
```

Comment:
```suggestion
    type.tp_doc = "GPI iterator handle.";
```
Not just subobject, this does loads and drivers too.